plugins/reach/lib/model/data/VendorDubbingCatalogItem.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Skeleton subclass for representing a row from the 'vendor_catalog_item' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements.  This class will only be generated as
+ * long as it does not already exist in the output directory.
+ *ac
+ * @package plugins.reach
+ * @subpackage model
+ */
+class VendorDubbingCatalogItem extends VendorCatalogItem
```

Comment:
in general since audio description and dubbing have almost every property in common worth having some parent class with the overlapping functions  and properties that both of the features can extend